docs/outlook/onmessagesend-onappointmentsend-events.md

```diff
@@ -137,7 +137,83 @@ If the **soft block** or **block** option is  used, the user can't send the item
+
+# [Web/Windows (new)](#tab/web-new-windows)
+
+For Outlook on the web and new Outlook on Windows, configure the **OnSendAddinsEnabled** mailbox policy in Exchange Online PowerShell.
+
+1. [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
+1. Create a new mailbox policy.
+
+   ```powershell
+    New-OwaMailboxPolicy OWAOnSendAddinAllUserPolicy
+   ```
+
+    > [!NOTE]
+    > Administrators can also use an existing policy.
+
+1. Set the **OnSendAddinsEnabled** flag to `true`.
+
+   ```powershell
+    Get-OwaMailboxPolicy OWAOnSendAddinAllUserPolicy | Set-OwaMailboxPolicy -OnSendAddinsEnabled:$true
+   ```
+
+1. Assign the policy to user mailboxes.
+
+   ```powershell
+    Get-User -Filter {RecipientTypeDetails -eq 'UserMailbox'} | Set-CASMailbox -OwaMailboxPolicy OWAOnSendAddinAllUserPolicy
+   ```
+
+# [Windows (classic)](#tab/windows)
+
+Configure the **Block send when web add-ins can't load** Group Policy setting. When the setting is turned on, mail items are moved to the **Drafts** folder when the **Send** button is selected. This way, when Outlook is able to load add-ins, any installed Smart Alerts add-ins can run checks on the items before they're sent.
+
+To turn on the setting, perform the following:
+
+1. Download the latest [Administrative Templates tool](https://www.microsoft.com/download/details.aspx?id=49030).
+1. Open the **Local Group Policy Editor** (**gpedit.msc**).
+1. Navigate to **User Configuration** > **Administrative Templates** > **Microsoft Outlook 2016** > **Security** > **Trust Center**.
+1. Open the **Block send when web add-ins can't load** setting.
+1. In the dialog that appears, select **Enabled**.
+1. Select **OK** or **Apply** to save your change.
+
+# [Mac](#tab/mac)
+
+For Outlook on Mac, the **OnSendAddinsWaitForLoad** mailbox key must be configured on each user's machine. This key ensures that add-ins are loaded from Exchange and are available to run checks on outgoing items. As the **OnSendAddinsWaitForLoad** key is CFPreference-compatible, it can be set by any enterprise management software for Mac, such as Jamf Pro. The following table provides details about the key.
+
+|Field|Value|
+|:---|:---|
+|**Domain**|com.microsoft.outlook|
+|**Key**|OnSendAddinsWaitForLoad|
+|**DataType**|Boolean|
+|**Possible values**|**false** (default): The currently downloaded manifests of the Smart Alerts add-ins (not necessarily the latest versions) run on outgoing mail items.<br><br>**true**: After the latest manifests of the Smart Alerts add-ins are downloaded from Exchange, the add-ins run on outgoing mail items. Otherwise, the item is blocked from being sent and the **Send** button becomes unavailable.|
```

Comment:
```suggestion
|**Possible values**|`false` (default): The currently downloaded manifests of the Smart Alerts add-ins (not necessarily the latest versions) run on outgoing mail items.<br><br>`true`: After the latest manifests of the Smart Alerts add-ins are downloaded from Exchange, the add-ins run on outgoing mail items. Otherwise, the item is blocked from being sent and the **Send** button becomes unavailable.|
```